docs/InstallGuide.md

```diff
@@ -131,294 +138,221 @@ To automatically launch the mysql server, execute the following command.
-    interface name: if-bvixz40n
-    mac address: 52:54:00:0a:fb:78
-    *** vm4 ***
-    node-id: 4
-    interface name: if-7st2alal
-    mac address: 52:54:00:e5:f9:fb
+```
+# yum -y install lxc lxc-templates
+```
 
-### Check the IP address of virtual machines
+Create and mount cgroup
 
-You can log into the virtual machines using the telnet command.
+```
+# mkdir /cgroup
+# echo "cgroup /cgroup cgroup defaults 0 0" >> /etc/fstab
+# mount /cgroup
+```
 
-    # telnet localhost PORT-NO
+Create 2 LXC guests
 
-The port numbers are:
+```
+# lxc-create -t centos -n inst1
+# lxc-create -t centos -n inst2
+```
 
-+ vm1 : 5001
-+ vm2 : 5002
-+ vm3 : 5003
-+ vm4 : 5004
+Configure interfaces of each guest
 
-The ip addresses are assigned as follows. As you can see, vm1 and vm3, share the same ip address, as do vm2 and vm4.
+```
+# vi /var/lib/lxc/inst1/config
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.veth.pair = inst1
+lxc.network.ipv4 = 10.100.0.10
+lxc.network.hwaddr = 10:54:FF:00:00:01
+lxc.rootfs = /var/lib/lxc/inst1/rootfs
+lxc.include = /usr/share/lxc/config/centos.common.conf
+lxc.arch = x86_64
+lxc.utsname = inst1
+lxc.autodev = 0
+```
 
-+ vm1 : 10.102.0.10
-+ vm2 : 10.102.0.10
-+ vm3 : 10.102.0.11
-+ vm4 : 10.102.0.11
+```
+# vi /var/lib/lxc/inst2/config
+lxc.network.type = veth
+lxc.network.flags = up
+lxc.network.veth.pair = inst2
+lxc.network.ipv4 = 10.100.0.11
+lxc.network.hwaddr = 10:54:FF:00:00:02
+lxc.rootfs = /var/lib/lxc/inst2/rootfs
+lxc.include = /usr/share/lxc/config/centos.common.conf
+lxc.arch = x86_64
+lxc.utsname = inst2
+lxc.autodev = 0
+```
 
-Log into the virtual machines and make sure their ip addresses are correct.
+Make sure that the IPv4 address and MAC address are the same as what you specify when you create the interface database records.
 
-**ex) vm1**
+Launch the LXC guests then enslave the tap interfaces to the datapath.
 
-    # telnet localhost 5001 <- vm1
-    Trying 127.0.0.1...
-    Connected to localhost.
-    Escape character is '^]'.
+```
+# lxc-start -d -n inst1
+# lxc-start -d -n inst2
 
-    CentOS release 6.4 (Final)
-    Kernel 2.6.32-358.el6.x86_64 on an x86_64
+# ovs-vsctl add-port br0 inst1
+# ovs-vsctl add-port br0 inst2
+```
 
-    vm1 login: root
-    Password:
-    Last login: Thu Nov 28 21:06:57 on ttyS0
-    [root@vm1 ~]# ip addr
-    1: lo: <LOOPBACK,UP,LOWER_UP> mtu 16436 qdisc noqueue state UNKNOWN
-        link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
-        inet 127.0.0.1/8 scope host lo
-        inet6 ::1/128 scope host
-           valid_lft forever preferred_lft forever
-    2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP qlen 1000
-        link/ether 52:54:00:0d:84:8c brd ff:ff:ff:ff:ff:ff
-        inet 10.102.0.10/24 brd 10.102.0.255 scope global eth0
-        inet6 fe80::5054:ff:fe0d:848c/64 scope link
-           valid_lft forever preferred_lft forever
+Log in to the inst1 to see if the IP address is assigned properly.
```

Comment:
I think we should clarify here that the IP address gets assigned by DHCP and that the container must send a DHCP request *after* the host has done its `ovs-vsctl add-port` command